internal/http/services/dataprovider/dataprovider.go

```diff
@@ -54,18 +53,6 @@ func New(m map[string]interface{}) (global.Service, error) {
 		return nil, err
 	}
 
-	if conf.Prefix == "" {
-		conf.Prefix = "data"
```

Comment:
We should provide a good default, exposing on root (/) is not good as it shadows all services in standalone mode.

What about `tus`?

Reply:
I think `data` is fine because it can do more than just tus.